jvm/src/main/kotlin/app/cash/trifle/delegate/EndEntityDelegate.kt

```diff
@@ -0,0 +1,14 @@
+package app.cash.trifle.delegate
+
+import app.cash.trifle.protos.api.alpha.MobileCertificateRequest
+
+interface EndEntityDelegate {
+  /**
+   * Create a mobile signing certificate request with a Trifle Content Signer.
+   *
+   * @param entityName the name with which we'll associate the public key.
+   */
+  fun createMobileCertRequest(
+    entityName: String,
+  ): MobileCertificateRequest
```

Comment:
this method actually shouldn't be in the API either (another thing I was going to clean up in CertificateUtil): it's returning a proto and clients shouldn't get proto results.  This can be lumped into the MobileCertificateRequest change though I think as a follow-on.